### PR TITLE
Add notifications counter

### PR DESCRIPTION
**Pull request by thierrymarianne**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add counter to modules notification tab
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/STAB-30
| How to test?  | Go to admin-dev/index.php/module/notifications and check notification tab presentation

### CODE REVIEW

**Comment by aleeks** [comment]
@supergb  is it ok for you ?!

File: src/Core/Addon/Module/ModuleManager.php [hunk 1]
```diff
@@ -91,6 +102,93 @@ public function __construct(AdminModuleDataProvider $adminModulesProvider,
     }
 
     /**
+     * @param callable $modulesPresenter
+     * @return object
+     */
+    public function getModulesWithNotifications(callable $modulesPresenter)
```
Comment (mickaelandrieu) [inline src/Core/Addon/Module/ModuleManager.php]: Why not type hint on ``ModulePresenter`` instead?

**Comment by superGB** [comment]
@aleeks Yes ok for me !

**Comment by aleeks** [comment]
Thank you @thierrymarianne

**Pull request merged**